reaktive/src/commonMain/kotlin/com/badoo/reaktive/observable/TakeWhilePredicate.kt

```diff
@@ -0,0 +1,33 @@
+package com.badoo.reaktive.observable
+
+import com.badoo.reaktive.base.tryCatch
+import com.badoo.reaktive.completable.CompletableCallbacks
+import com.badoo.reaktive.disposable.Disposable
+import com.badoo.reaktive.disposable.DisposableWrapper
+
+/**
```

Comment:
Let's just copy the text from [here](http://reactivex.io/documentation/operators/takewhile.html). And put the link as well: 
`See: [http://reactivex.io/documentation/operators/takewhile.html]`

Reply:
Done:

```kotlin
/**
 * Mirror items emitted by upstream until a specified condition becomes false.
 * See: [http://reactivex.io/documentation/operators/takewhile.html].
 */
```

The original doc was copied from RxJava, but this reads better.